README.md

```diff
@@ -82,15 +82,6 @@ Check the below video for a quick demo of XC3.
    Note : if no private/public subnets provided then XC3 will create new VPC, private and public subnets and also XC3 will destroy these resources once
    user destroys XC3 setup.
 
-10. XC3 will create an EC2 instance during deployment, the user needs to create an AWS key_pair file in order to login to EC2 instance for troubleshooting purpose.
-11. If the ssh access is restricted only through bastion/jump server/vpn, user should have the security group ID of the bastion/jump/vpn EC2 instance.
-12. The user has to **enable CostExplorer** by following the below link.
-
-    https://docs.aws.amazon.com/awsaccountbilling/latest/aboutv2/ce-enable.html
```

Comment:
why are we removing these lines

Reply:
This line is repeated so, removed it.
![image](https://github.com/XgridInc/xc3/assets/129849123/8a80de63-75a9-415f-a027-250fcb5b279e)